docs/spec/api.md

```diff
@@ -1013,7 +1013,8 @@ The response will be in the following format:
     }
 
 For repositories with a large number of tags, this response may be quite
-large. If such a response is expected, one should use the pagination.
```

Comment:
You're absolutely right, I've made a huge mistake. I'm closing this PR down because I read the following:

`A registry may also limit the amount of responses returned even if pagination was not explicitly requested.`

This documentation is related to the API, not to the implementation itself, so it doesn't make sense to have the information here.